flash/core/utilities/apply_func.py

```diff
@@ -38,4 +39,4 @@ def _is_overridden(method_name: str, instance: object, parent: Type[object]) ->
     if not hasattr(instance, method_name):
         return False
 
-    return getattr(instance, method_name).__code__ != getattr(parent, method_name).__code__
+    return is_overridden(method_name, instance, parent)
```

Comment:
I think the `_is_overridden` function could be entirely replaced by the `lightning_utilities` version